setup.py

```diff
@@ -72,6 +72,11 @@ def run(self):
             'pytest>=6.0',
             'pytest-cov',
             'freezegun~=1.0',
+			"tzdata;sys_platform == 'win32'",
+			'backports.zoneinfo; python_version<"3.9"',
+			'pytz',
+			'setuptools',
+			'jinja2>=3.0',
```

Comment:
There's some extra space:

```suggestion
            'freezegun~=1.0',
	    "tzdata;sys_platform == 'win32'",
	    'backports.zoneinfo; python_version<"3.9"',
	    'pytz',
	    'setuptools',
	    'jinja2>=3.0',
```